cognite/neat/graph/stores/_memory_store.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+from rdflib import Graph, Namespace
+
+from cognite.neat.constants import DEFAULT_NAMESPACE, PREFIXES
+
+from ._base import NeatGraphStoreBase
+
+
+class MemoryStore(NeatGraphStoreBase):
+    """MemoryStore is a class that stores the graph in memory and provides methods to read/write data it contains.
```

Comment:
```suggestion
    """MemoryStore is a class that stores the graph in memory using rdflib and provides methods to read/write data it contains.
```

Reply:
Specific info on MemoryStore:
https://rdflib.readthedocs.io/en/stable/apidocs/rdflib.plugins.stores.html#rdflib.plugins.stores.memory.Memory

Offtopic: Reading [this](https://rdflib.readthedocs.io/en/stable/persistence.html#stores-currently-shipped-with-core-rdflib), I am thinking should we try to create FDMStore rdflib plugin for `FMD`. This would make `neat` even more attractive.